g11n-ws/vip-common/src/main/java/com/vmware/vip/common/i18n/status/APIResponseStatus.java

```diff
@@ -33,4 +33,6 @@ public class APIResponseStatus {
             "Not Found");
     public static final Response UNKNOWN_ERROR = new Response(520, "Unknown Error");
     public static final Response TRANSLATION_COLLECT_REQUEST_SUCCESS = new Response(200, "Source collection request have received");
+    public static final Response TRANSLATION_COLLECT_SOURCE_PARAM_IS_FAlSE = new Response(200, "Received the sources and comments(please use translation-product-component-api to confirm it).");
```

Comment:
Is this message correct? When 'collectSource=false', still can receive the sources?

Reply:
No, It just let client use the  translation-product-component-api to confirm it